Sources/Sentry/SentryThreadInspector.m

```diff
@@ -33,6 +38,12 @@ - (id)initWithStacktraceBuilder:(SentryStacktraceBuilder *)stacktraceBuilder
 
     int threadCount = [self.machineContextWrapper getThreadCount:context];
 
+    thread_act_array_t suspendedThreads = nil;
+    mach_msg_type_number_t numSuspendedThreads = 0;
+    if (forceStackTraces) {
+        sentrycrashmc_suspendEnvironment(&suspendedThreads, &numSuspendedThreads);
+    }
```

Comment:
Ah gosh, I was close 2 years ago to get the stacktraces for all threads, but we wanted to ship something fast and didn't want to block all threads when capturing a message. Anyways, we could consider enabling this for all capture methods. Maybe with an option. I created an issue https://github.com/getsentry/sentry-cocoa/issues/1870

Reply:
Yes, we should investigate this option